content/spin/install.md

```diff
@@ -166,7 +166,7 @@ This does not install any Spin templates or plugins. For a starter list, see the
 ## Installing Templates and Plugins
```

Comment:
Can we add a line that says users must be on Spin v1.3 or later to use this feature?